### ISSUE TITLE
Concat dependency annoyance

### ISSUE THREAD

**Issue by den-is** [justindowning/puppet-awscli#54]
Hey,

Can you please update your module on forge.puppet.com ???

```
Warning: Module 'puppetlabs-concat' (v2.2.0) fails to meet some dependencies:
  'jdowning-awscli' (v1.3.0) requires 'puppetlabs-concat' (>= 1.0.0 <2.0.0)
```

1. Here on github in metadata.json you have
```
{ "name": "puppetlabs/concat", "version_requirement": ">= 1.0.0 <3.0.0" }
```

2. In module from forge.puppet.com, same version module v1.3.0 has different metadata.json
```
{"name":"puppetlabs/concat","version_requirement":">= 1.0.0 <2.0.0"}
```

Also concat is [v4.0.0](https://forge.puppet.com/puppetlabs/concat) now so prabably dependecy needs to be updated to `<5.0.0`

**Comment by justindowning** [comment]
I'm actively working on puppet 4 compatibility (which requires the latest version of puppet-concat) in #51.

**Comment by justindowning** [comment]
@den-is This should be fixed in 128e3b6. Let me know if you have any issues.

**Comment by den-is** [comment]
so far so good - almost 8 hours in prod
LGTM
hope to see at least fixed v1.3**.1** in forge soon O:)
as i'm using CodeManager for deploys, i'm trying to avoid external module installation not from forge.

**Comment by justindowning** [comment]
Awesome! I'm thinking with all the changes since 1.3, next release should probably be 2.0 (especially considering puppet 4 support).
